docs/core/deploying/trim-warnings/il2056.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "IL2056: A 'System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute' annotation on a property conflicts with the same attribute on its backing field"
+description: "Learn about trim warning IL2056: DynamicallyAccessedMembersPropertyConflictsBackingField"
+ms.date: 07/23/2021
+ms.topic: reference
+author: mateoatr
+ms.author: matorre
+f1_keywords:
+  - "IL2056"
+---
+# IL2055: Call to 'System.Type.MakeGenericType' cannot be statically analyzed by the trimmer
```

Comment:
This was copy pasted from IL2055 - needs to be changed to "Trim analysis: 'DynamicallyAccessedMemberAttribute' on property 'property' conflicts with the same attribute on its backing field 'field'."